### ISSUE TITLE
Setting pod_security_policy_template_id does not work

### ISSUE THREAD

**Issue by rbq** [rancher/terraform-provider-rancher2#167]
Setting `pod_security_policy_template_id` on projects (introduced with PR #82) doesn't seem to work (but reading does). I use my own PSP (not one of the two built-in ones) and get the addition shown in the diff over and over. The change disappears only after I manually change it from „None“ to my custom one via Rancher UI.

**Comment by rawmind0** [comment]
Submitted PR #175 to fix issue.

**Comment by rawmind0** [comment]
PR is merged into master. Please, reopen issue if needed.

**Comment by llienard** [comment]
Hi, it doesn't seem to work anymore on project creation (but updating does).

```
# versions.tf
terraform {
  required_providers {
    rancher2 = {
      source = "rancher/rancher2"
      version = "~> 1.10.0"
    }
  }
  required_version = ">= 0.13"
}

# main.tf
data "rancher2_pod_security_policy_template" "my_pod_security_policy" {
  name = "my_pod_security_policy"
}

resource "rancher2_project" "my_project" {
  cluster_id = "cluster-id"
  name = "my-project"
  pod_security_policy_template_id = data.rancher2_pod_security_policy_template.my_pod_security_policy.id
}
```

**Comment by rawmind0** [comment]
Thanks for reporting @llienard . PR #432 should address the issue
